Data/CubeBlocks.sbc

```diff
@@ -2,11 +2,91 @@
 <Definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <CubeBlocks>
 
+  <!-- Begin modded beacon -->
+  
+  <Definition>
+      <Id>
+        <TypeId>Beacon</TypeId>
+        <SubtypeId>LargeBlockBeacon</SubtypeId>
+      </Id>
+      <DisplayName>DisplayName_Block_Beacon</DisplayName>
```

Comment:
What's the purpose of this commented-out xml? How were you attempting to change the beacon that didn't work, and how is that information contained here? If this is highly similar to the other blocks below I'd prefer to just list what we change to adapt to a "modified" beacon instead of repeating it all.